crates/bevy_reflect/src/impls/std.rs

```diff
@@ -1516,9 +1516,13 @@ mod tests {
     };
     use bevy_utils::{Duration, Instant};
     use bevy_utils::{EntityHashMap, HashMap};
+    use static_assertions::assert_impl_all;
     use std::f32::consts::{PI, TAU};
     use std::path::Path;
 
+    // EntityHashMap should implement Reflect
+    assert_impl_all!(EntityHashMap<i32, i32>: Reflect);
```

Comment:
shouldn't this still be in a test function? like the one before: `entity_hashmap_should_impl_reflect`

Reply:
Other static_assertions in bevy are similarly not inside a test function: https://github.com/bevyengine/bevy/pull/11182/files#diff-99929377bff6c8e2987d61c8887213b22703c4e62e3d551d3243a1ecbf21869fR425